mllib/src/main/scala/org/apache/spark/ml/feature/PolynomialExpansion.scala

```diff
@@ -46,7 +46,7 @@ class PolynomialExpansion(override val uid: String)
    * @group param
    */
   val degree = new IntParam(this, "degree", "the polynomial degree to expand (>= 1)",
-    ParamValidators.gt(1))
+    ParamValidators.gtEq(1))
```

Comment:
This has been done in https://github.com/apache/spark/pull/11216, and I think that would be merged shortly (also has a related test case addition).